massmigration/migrations.py

```diff
@@ -33,6 +45,25 @@ class BaseMigration:
     # this attribute.
     backend_method: str = None
 
+    def __init_subclass__(cls, **kwargs):
+        super().__init_subclass__(**kwargs)
+        all_db_aliases = get_all_db_aliases()
+
+        if (
+            cls.allowed_db_aliases is not None and  # If not set, then all dbs are allowed
+            any([allowed_db not in all_db_aliases for allowed_db in cls.allowed_db_aliases])
+        ):
+            raise DbAliasNotAllowed(
+                    f"Provided Migration <allowed_database_alias> for {self.key} Migration are invalid."
```

Comment:
```suggestion
                    f"Migration {self.key} provided invalid value(s) in `allowed_database_aliases`. "
```

Just fixing a typo in `<allowed_database_alias>`, tweaking wordingly slightly, and adding a space at the end.